client/client_test.go

```diff
@@ -231,7 +235,7 @@ func TestClientRunTransaction(t *testing.T) {
 
 	for _, commit := range []bool{true, false} {
 		value := []byte("value")
-		key := []byte(fmt.Sprintf("key-%t", commit))
+		key := []byte(fmt.Sprintf("%s/key-%t", testUser, commit))
```

Comment:
it's convenient not to need an extra const related to it. Ultimately, we'll need namespace/database/table for these kinds of things.